makefile

```diff
@@ -219,6 +219,18 @@ doc-example-names = $(example-names:%=doc/examples/%.html)
 
 doc-lib-names = $(lib-names:%=doc/lib/%.html)
 
+tutorial-data = t10k-images-idx3-ubyte t10k-labels-idx1-ubyte
+tutorial-data := $(tutorial-data:%=examples/%)
+
+$(tutorial-data):
```

Comment:
Normally, putting multiple outputs on the LHS of a `:` in make is a bit of a foot-gun, but it's exactly what we want here. When you have "multiple" outputs and it's not a pattern (`%`) rule, Make creates one fresh rule per output, with the same dependencies and commands as in the original definition. When it _is_ a pattern rule, then Make believes that the single set of commands will generate _all_ of the listed files (i.e. true multi-output).

My instinct is normally to use pattern rules since their behavior is less surprising.